Workbooks/Workloads/Redis Insights/Redis Insights.workbook

```diff
@@ -285,11 +282,11 @@
+              "query": "let data = InsightsMetrics\r\n| where (Computer in ({OnboardedComputers}) or '*' in ({OnboardedComputers})) and Name in~ ('total_connections_received','total_commands_processed','calls')\r\n| order by Computer asc, Name asc, Tags asc, TimeGenerated asc\r\n| extend Value = iff(Computer == prev(Computer) and Name == prev(Name) and Tags == prev(Tags) and Val > prev(Val), Val - prev(Val), real(0));\r\ndata| order by Computer asc, Name asc, TimeGenerated asc\r\n| summarize \r\n        ReceivedConnections = sumif(Value, Name == 'total_connections_received'),\r\n        ReceivedConnectionsRate = sumif(Value, Name == 'total_connections_received') / (({TimeRange:end} - {TimeRange:start}) / 1s),\r\n        Commands = sumif(Value, Name == 'calls'),\r\n        CommandsRate = sumif(Value, Name == 'calls') / (({TimeRange:end} - {TimeRange:start}) / 1s)\r\n    by Computer\r\n| join kind = inner (data\r\n| where Name == 'total_connections_received'\r\n| make-series ConnectionTrend = sum(Value) default = 0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by Computer) on Computer\r\n| join kind = inner (InsightsMetrics \r\n    | where (Computer in ({OnboardedComputers}) or '*' in ({OnboardedComputers})) \r\n    and Namespace == \"Processor\" and Name == \"UtilizationPercentage\"     \r\n    | summarize Cpu = avg(Val) by Computer\r\n    ) on Computer\r\n| join kind = inner (InsightsMetrics \r\n    | where (Computer in ({OnboardedComputers}) or '*' in ({OnboardedComputers})) \r\n    and Namespace == \"Memory\" and Name == \"AvailableMB\" \r\n    | extend TotalMemory = toreal(todynamic(Tags)[\"vm.azm.ms/memorySizeMB\"])\r\n    | extend MemoryUsedPercentage = 100 - (toreal(Val) / TotalMemory) * 100.0 \r\n    | summarize MemoryUtilizationPercent = avg(MemoryUsedPercentage) by Computer\r\n    ) on Computer    \r\n| project Computer, ReceivedConnections,ReceivedConnectionsRate, Commands, CommandsRate, Cpu, MemoryUtilizationPercent\r\n| join kind = rightouter (datatable(Computer:string)[{OnboardedComputers}]) on Computer\r\n| extend Computer = Computer1\r\n| project-away Computer1\r\n",
               "size": 0,
               "showAnalytics": true,
               "timeContext": {
-                "durationMs": 0
+                "durationMs": 14400000

```

Comment:
Its weird that the time context value changed even though it is tied to the TimeRange param in the line below.

Reply:
@gardnerjr, do you know why this happened?